src/tokens/tokens.service.ts

```diff
@@ -370,4 +437,20 @@ class TokenListener implements EventListener {
       };
     }
   }
+  private postOptions(operator: string) {
+    const from = `${this.service.shortPrefix}-from`;
+    const sync = `${this.service.shortPrefix}-sync`;
+    const call = `${this.service.shortPrefix}-call`;
+
+    const requestOptions: AxiosRequestConfig = {
+      params: {
+        [from]: operator,
+        [sync]: 'false',
+        [call]: 'true',
```

Comment:
I know you only have one usage right now and it is a `call`, but it seems like maybe this should be passed in as a bool for readability/flexibility.

Reply:
done